_posts/blog/2018-09-21/2018-09-21-3-things-that-make-it-really-easy-to-shop-in-openbazaar.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: "3 Things That Make It <span class='post-title-extra'>Really Easy to Shop</span> in OpenBazaar"
+layout: post
+date: '2018-09-21 06:30:00 -0300'
+social_title: '3 Things That Make It Really Easy to Shop in OpenBazaar'
+social_description: 'OpenBazaar is a decentralized online marketplace. OB1, the team leading the development, has designed a few things that make it easier to use.'
+social_image: Search Providers in OpenBazaar.gif.png
+---
+
+OpenBazaar is not like a regular online marketplace.
+
+It’s a _decentralized_ online marketplace.
+
+OpenBazaar is like its own separate part of the internet. It’s a network of users who connect directly to one another instead of to centralized websites like Amazon or Ebay. 
+
+The decentralized architecture that makes it powerful can also make it a bit hard to understand at first. 
+
+- [Why does it have to be downloaded?](https://openbazaar.org/blog/why-do-i-have-to-download-openbazaar/) 
+
+- [Why does it only use cryptocurrency for payments?](https://openbazaar.zendesk.com/hc/en-us/articles/207678086) 
+
+- [How do the different payment types work?](https://openbazaar.zendesk.com/hc/en-us/articles/360000303371-What-are-the-different-payment-types-in-OpenBazaar-)
+
+[OB1](https://ob1.io), the team leading the development of OpenBazaar, has designed a few things that make it easier to use.
+
+> Note that these items are add-ons and are [less decentralized](https://openbazaar.org/blog/the-beginners-guide-to-decentralization/) than the network itself. They make shopping easier but they do not change the decentralized protocol nor the privacy or security of your transactions.
+
+### 1. Easy discovery
+
+![Search Providers in OpenBazaar](Search Providers in OpenBazaar.gif "Search Providers in OpenBazaar")
+
+There are [search engines](https://openbazaar.org/blog/decentralized-search-and-content-discovery-on-openbazaar/) within the application that help you discover items. OB1 created the first search engine available within OpenBazaar but anyone else is welcome to build and add their own.
```

Comment:
They aren't 100% able to add their own. They'd have to ask us to include them as a default, and then they'd have to stick to our rules (which we aren't enforcing on BlockBooth currently).

They can, however, create a search engine and allow individual users to add it to their individual installation.

Reply:
Simplified it to just "anyone else is welcome to build one and add it to their own client."